frameworks/digital-outcomes-and-specialists/questions/briefs/specialistRole.yml

```diff
@@ -49,6 +49,11 @@ options:
       Build software that supports user needs. Continually improve the service by
       identifying new tools and techniques, removing technical bottlenecks, and adapting and maintaining code.
   -
+    label: Performance analytist
```

Comment:
Spelling... `Performance analyst`

Also for the value (Unless we saved it like that in the first place?)